workflow/snakemake_rules/main_workflow.smk

```diff
@@ -440,6 +440,7 @@ rule combine_samples:
     benchmark:
         "benchmarks/combine_samples_{build_name}.txt"
     conda: config["conda_environment"]
+    threads: 2
```

Comment:
> This change is optimized for Nextstrain's scheduled open and gisaid runs.

Unclear if this might slow down other people's workflow runs. If we only want to optimize this for the Nextstrain automated runs, we can set the job threads per profile (e.g. [custom thread for tree](https://github.com/nextstrain/ncov/blob/137ffc479a1bfe4e2c6ae1bbcd1b22e3afc5f8dd/nextstrain_profiles/nextstrain-gisaid/config.yaml#L10))

Reply:
Good idea, done in d53568d3b634b036410518e6e32ba0e3f6894bfe.